src/game-mode.ts

```diff
@@ -164,14 +164,14 @@ export class GameMode implements GameModeConfig {
     if (waveIndex % 10 !== 1 && waveIndex % 10) {
```

Comment:
```suggestion
    if (waveIndex % 10 > 1) {
```
Like what is this lmao